rfcs/text/semantic-release.md

```diff
@@ -0,0 +1,257 @@
+to this I believe they were written and updated by each collaborator.
+
+[`semantic-release`](https://github.com/semantic-release/semantic-release) has a package called [`@semantic-release/changelog`](https://github.com/semantic-release/changelog) when used in conjunction with [`@semantic-release/git`](https://github.com/semantic-release/git) and
+[`@semantic-release/npm`](https://github.com/semantic-release/npm) it will maintain a changelog that is committed to git and published to [NPM](https://www.npmjs.com/). We will use this to help
+communicate the features that each release brings. The changelog will include the `TYPE`, `SCOPE` and `DESCRIPTION`, it
+will not include the `BODY` but it will include a link to the relevant pull request.
+
+## Maintenance Releases
+The beta version of [`semantic-release`](https://github.com/semantic-release/semantic-release) includes extensive support for publishing a maintenance
+release. There is a [detailed walk through](https://github.com/semantic-release/semantic-release/blob/916c2685c57f3490fb1e50afbf72ea8dce11e188/docs/recipes/maintenance-releases.md)
+outlining many scenarios.
+
+This functionality is not available in the latest stable release, however the beta has been evolving for in excess of a year.
+
+
+# Drawbacks
+- There is a learning curve to this new process
+- Version numbers will increase substantially if we don't plan our interfaces carefully to prevent breaking changes. I
+don't consider this to be a drawback, but we need to be aware about breaking changes. Previously we have bundled many
+breaking changes into one release.
+- Changelog format will change
+
+# Alternatives
+There are alternatives to [`semantic-release`](https://github.com/semantic-release/semantic-release) for automating the
+release process such as [np](https://github.com/sindresorhus/np)
+
+However, I could find no other solution that parses commits to work out the version number. This means that someone has
+to be involved in the release process to determine the version number. This is not sustainable and prone to errors.
+
+# Adoption strategy
+
+This RFC PR includes all changes necessary, if this RFC is accepted when it is merged it will immediately start
+releasing from the following branches `['+([0-9])?(.{+([0-9]),x}).x', 'master', 'next', 'next-major', {name: 'beta', prerelease: true}, {name: 'alpha', prerelease: true}]`  
+[[Source]](https://github.com/semantic-release/semantic-release/blob/916c2685c57f3490fb1e50afbf72ea8dce11e188/docs/usage/configuration.md#branches)
+
+Adding [`probot/semantic-pull-requests`](https://probot.github.io/apps/semantic-pull-requests/) is a manual step and
+will have to be performed by a workspace administrator.
+
+# How we teach this
+
+The proposed tooling is well established in the JavaScript community. For carbon developers
+ can use `npm run commit` to launch [`commitizen`](https://github.com/commitizen/cz-cli) which will guide them through the process.
+![](./commitizen.svg)
```

Comment:
This SVG doesn't seem to be rendering here:

* https://github.com/Sage/carbon/blob/semantic-release/rfcs/text/semantic-release.md#how-we-teach-this

GitHub may not understand the `./` in the filepath. Maybe this would work:
```suggestion
![](rfcs/text/semantic-release/commitizen.svg)
```

Reply:
It doesn't work at the review stage but it will once committed. I tested on a private repo first.